lib/tasks/batch_connect.rake

```diff
@@ -0,0 +1,31 @@
+namespace :batch_connect do
+  desc "Generate new batch connect session"
+  task new_session: :environment do
+    # Read in user settings
+    app = ENV["BC_APP_TOKEN"] || abort("Missing environment variable BC_APP_TOKEN")
+    fmt = ENV["BC_RENDER_FORMAT"]
+    ctx = if !$stdin.tty?
```

Comment:
That would cause the rake task to always try to read from stdin. If a user just wanted to submit a job with default values, he'd have to remember to pipe in `"{}"` or the rake task will just hang waiting for input.